### ISSUE TITLE
Route table route count

### ISSUE THREAD

**Issue by rayhwang-kcom** [k1LoW/awspec#341]
Hi,

I've just added this to `awspec/lib/awspec/type/route_table.rb:38`: 

```
    def route_count
      resource_via_client.routes.count
    end
```

Is it possible to add merge this into the code so that we can test the route count? 

Thanks.

**Comment by k1LoW** [comment]
Hi @rayhwang-kcom 

> Is it possible to add merge this into the code so that we can test the route count?

Yes! This enhancement like https://github.com/k1LoW/awspec/pull/336 .

**Comment by k1LoW** [comment]
I'm looking forward to your PR 👍

**Comment by rayhwang-kcom** [comment]
Pull request open: https://github.com/k1LoW/awspec/pull/342

**Comment by k1LoW** [comment]
Released as v1.1.0.
